.github/workflows/build-and-release-edge.yml

```diff
@@ -5,11 +5,83 @@ on:
     branches:
       - "master"
     paths-ignore:
-      - "docs/**"
       - ".github/**"
```

Comment:
```suggestion
      - "docs/**"
      - ".github/**"
```

Reply:
If we don't remove docs from paths-ignore I think it will not be able to track the changes in docs directory so there will be no commits made